src/app/pages/uploads/upload-image/upload-image.component.scss

```diff
@@ -0,0 +1,178 @@
+@import "../../../shared-styles/all-sass-imports.scss";
```

Comment:
You can make angular resolve style imports by adding this to your angular.json

`
            "stylePreprocessorOptions": {
              "includePaths": ["src/app/shared-styles"]
            },`
            
            and then you can do in your scss file
            `@import  "all-sass-imports"`
            you can rename it to main.scss or something and import "main"